retnet/modeling_retnet.py

```diff
@@ -522,24 +736,16 @@ def forward(self, x):
     def extra_repr(self):
         return "p={}".format(self.drop_prob)
 
-
 class RetNetDecoderLayer(nn.Module):
 
     def __init__(self, config: RetNetConfig, depth: int, tensor_parallel: bool = False):
         super().__init__()
         self.config = config
         self.embed_dim = config.decoder_embed_dim
         self.dropout_module = torch.nn.Dropout(config.dropout)
+        self.drop_path = DropPath(np.linspace(0, config.drop_path_rate, config.decoder_layers)[depth]) if config.drop_path_rate > 0 else None
```

Comment:
I prefer previous code. This one-liner is too long and breaks the 100 character limit.